uwrt_mars_rover_controllers/uwrt_open_loop_position_controller/include/position_controllers/joint_group_position_controller.h

```diff
@@ -0,0 +1,114 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2008, Willow Garage, Inc.
+ *  Copyright (c) 2012, hiDOF, Inc.
+ *  Copyright (c) 2013, PAL Robotics, S.L.
+ *  Copyright (c) 2014, Fraunhofer IPA
+ *  All rights reserved.
```

Comment:
add ur name to copyright or add uw robotics and the current year. (as long as youve modified 3rd party code)